spotipy/client.py

```diff
@@ -386,6 +386,19 @@ def artist_albums(
                 - country - limit the response to one particular country.
                 - limit  - the number of albums to return
                 - offset - the index of the first album to return
+
+            Response:
+                - Type: Dictionary
+
+            Example Code Output:
+                Young Heart
+                Beautiful Lies
+                Beautiful Lies
+                Beautiful Lies (Deluxe)
+                Beautiful Lies (Deluxe)
+                Fire Within
+                Fire Within
+                Fire Within (Deluxe)
```

Comment:
I believe it's a bit unusual to provide example code outputs in python libraries, no real need for it